index.html

```diff
@@ -69,24 +71,531 @@ <h3 class="main-subheading">
         <img src="#" alt="OtherImg" class="fourth-illustration" />
       </figure>
       <section class="product-container">
+        <h2 class="hidden-heading">Collection Categories</h2>
         <ul class="product-list">
           <li class="product-item">
             <img src="#" alt="ProductImg" class="product-img" />
+            <a href="#" class="collection-link">UI Kits</a>
           </li>
           <li class="product-item">
             <img src="#" alt="ProductImg" class="product-img" />
+            <a href="#" class="collection-link">Social Media</a>
           </li>
           <li class="product-item">
             <img src="#" alt="ProductImg" class="product-img" />
+            <a href="#" class="collection-link">Presentation</a>
           </li>
           <li class="product-item">
             <img src="#" alt="ProductImg" class="product-img" />
+            <a href="#" class="collection-link">Fonts</a>
           </li>
           <li class="product-item">
             <img src="#" alt="ProductImg" class="product-img" />
+            <a href="#" class="collection-link">Illustrations</a>
+          </li>
+        </ul>
+      </section>
+      <section class="spotlight-container">
+        <div class="spotlight-header">
+          <h2 class="spotlight-heading">Spotlight Digital Assets Selections</h2>
+          <a href="#" class="explore-featured">Explore Featured Works &gt;</a>
+        </div>
+        <div class="spotlight-product-container">
+          <article class="spotlight-product">
+            <h2 class="hidden-heading">Digital Assets</h2>
+            <img src="#" alt="SpotlightProduct" />
+            <div class="spotlight-product-details">
+              <a class="spotlight-product-title">Flexsoul</a>
+              <p class="spotlight-price">$12.00</p>
+              <ul class="spotlight-tags">
+                <li class="spotlight-tag">
+                  <p class="spotlight-tag-text">Yoga Studio</p>
+                </li>
+                <li class="spotlight-tag">
+                  <img src="#" alt="TagSpacer" class="spotlight-tag-spacer" />
```

Comment:
A list is an enumeration of homogeneous elements with the same meaning and styling. This means that any number of items can be added to the list, and they should all look the same and have the same markup.
So this is not a list item, it will be a styling element (pseudo element in css) we will talk about this later.